testing/state-transition/state-transition.go

```diff
@@ -118,9 +119,11 @@ func SetupTestState(t *testing.T, cs chain.Spec) (
 
 	dummyProposerAddr := []byte{0xff}
 
-	kvStore, depositStore, err := initTestStores()
+	cms, kvStore, depositStore, err := BuildTestStores()
 	require.NoError(t, err)
-	beaconState := statedb.NewBeaconStateFromDB(kvStore, cs)
+
+	ms := sdk.NewContext(cms.CacheMultiStore(), true, log.NewNopLogger())
+	beaconState := statedb.NewBeaconStateFromDB(kvStore.WithContext(ms), cs)
```

Comment:
this beaconState is basically what `storage.StateFromContext` does.
I should consider refactoring it out to make it clear